src/test/scala/org/ergoplatform/nodeView/history/BlockSectionValidationSpecification.scala

```diff
@@ -31,14 +31,14 @@ class BlockSectionValidationSpecification extends HistoryTestHelpers {
     // checks, specific for extension
```

Comment:
checks for interlinks correctness should be added here

Reply:
@catena2w, done